content/blog/2017-08-23-ml-model-ensembling-with-fast-iterations.md

```diff
@@ -197,7 +197,7 @@ process for this project
 `gist:gvyshnya/7f1b8262e3eb7a8b3c16dbfd8cf98644#dvc.bat`
 
 If you then further edit ensemble configuration setup in `code/config.R`, you
-can simply leverage the power of DVC as for automatic dependencies resolving and
+can leverage the power of DVC as for automatic dependencies resolving and
```

Comment:
Rather not change old blog posts, especially from external authors. But thanks! Please revert this one.

Reply:
This isn't resolved. Unresolving.

Reply:
Oh never mind I see this one is rolled back. Rexolving...